### PR TITLE
chore(project): live stream cards to links

### PR DESCRIPTION
**Pull request by olga-jwp**
## Description

This PR solves # .

### Steps completed:

According to our definition of done, I have completed the following steps:

- [ ] Acceptance criteria met
- [ ] Unit tests added
- [ ] Docs updated (including config and env variables)
- [ ] Translations added
- [ ] UX tested
- [ ] Browsers / platforms tested
- [ ] Rebased & ready to merge without conflicts
- [ ] Reviewed own code

### CODE REVIEW

File: src/pages/ScreenRouting/playlistScreens/PlaylistLiveChannels/PlaylistLiveChannels.tsx [hunk 1]
```diff
@@ -24,12 +24,13 @@ import { generateMovieJSONLD } from '#src/utils/structuredData';
...
 import type { Playlist } from '#types/playlist';
 import Loading from '#src/pages/Loading/Loading';
+import VideoDetails from '#components/VideoDetails/VideoDetails';
 
 const PlaylistLiveChannels: ScreenComponent<Playlist> = ({ data: { feedid, playlist } }) => {
   const { t } = useTranslation('epg');
 
   // Config
-  const { config, accessModel } = useConfigStore(({ config, accessModel }) => ({ config, accessModel }), shallow);
+  const { config } = useConfigStore(({ config, accessModel }) => ({ config, accessModel }), shallow);
```
Comment (budzworthy) [inline src/pages/ScreenRouting/playlistScreens/PlaylistLiveChannels/PlaylistLiveChannels.tsx]:
```suggestion
  const { config } = useConfigStore(({ config }) => ({ config }), shallow);
```

File: src/pages/ScreenRouting/playlistScreens/PlaylistLiveChannels/PlaylistLiveChannels.tsx [hunk 2]
```diff
@@ -185,25 +179,16 @@ const PlaylistLiveChannels: ScreenComponent<Playlist> = ({ data: { feedid, playl
...
-        primaryMetadata={primaryMetadata}
-        image={videoDetails.image}
-        startWatchingButton={startWatchingButton}
-        shareButton={shareButton}
-        trailerButton={null}
-        favoriteButton={null}
-        getURL={() => ''} // TODO: OWA-31 - Add link for EPG item
-        player={
-          channelMediaItem && (
+      <div className={styles.videoCinemaLayout} data-testid={testId('cinema-layout')}>
```
Comment (budzworthy) [inline src/pages/ScreenRouting/playlistScreens/PlaylistLiveChannels/PlaylistLiveChannels.tsx]: Can you put this `div` inside of `VideoDetails` so it's not duplicated here and in `VideoLayout`? You can also then move the `videoCinemaLayout` style from `VideoLayout` to `VideoDetails`.

File: src/pages/ScreenRouting/playlistScreens/PlaylistLiveChannels/PlaylistLiveChannels.tsx [hunk 3]
```diff
@@ -215,20 +200,20 @@ const PlaylistLiveChannels: ScreenComponent<Playlist> = ({ data: { feedid, playl
...
-            channel={channel}
-            program={program}
-            config={config}
-          />
-        </div>
-      </VideoLayout>
+          )}
+        </VideoDetails>
+      </div>
+      <div className={styles.epgContainer}>
```
Comment (budzworthy) [inline src/pages/ScreenRouting/playlistScreens/PlaylistLiveChannels/PlaylistLiveChannels.tsx]: Should this be inside of VideoDetails so it stays the same as it was?

File: src/pages/ScreenRouting/playlistScreens/PlaylistLiveChannels/PlaylistLiveChannels.tsx [hunk 4]
```diff
@@ -110,20 +111,13 @@ const PlaylistLiveChannels: ScreenComponent<Playlist> = ({ data: { feedid, playl
     setActiveChannel(channelId);
   };
 
-  // Effects
```
Comment (budzworthy) [inline src/pages/ScreenRouting/playlistScreens/PlaylistLiveChannels/PlaylistLiveChannels.tsx]: We might want to leave this. I'm not sure if there's any EPG auto-advance stuff that may break without it or if there are other incoming links that rely on it.

File: src/components/VideoDetails/VideoDetails.tsx [hunk 5]
```diff
@@ -54,6 +56,7 @@ const VideoDetails: React.VFC<Props> = ({
           </div>
         </div>
       </div>
+      {children}
```
Comment (budzworthy) [inline src/components/VideoDetails/VideoDetails.tsx:61]: This should move outside of the div `<div className={styles.video} data-testid={testId('video-details')}>` (i.e. to line 61)

Then wrap both divs with the div `<div className={styles.videoCinemaLayout} data-testid={testId('cinema-layout')}>`

**Review by budzworthy — CHANGES_REQUESTED** [review]
I made a few suggestions to try to keep the html as exactly the same as possible to what it was previously.

File: src/pages/ScreenRouting/playlistScreens/PlaylistLiveChannels/PlaylistLiveChannels.tsx (hunk 2 above)
Comment (olga-jwp) [reply src/pages/ScreenRouting/playlistScreens/PlaylistLiveChannels/PlaylistLiveChannels.tsx]: It looks like there are no styles attached to the `videoCinemaLayout`, so I'll move it inside `VideoDetails` for the `data-testid `

**Review by budzworthy — APPROVED** [review]

**Pull request merged**